lib/services/locator/locator_service.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:get_it/get_it.dart';
+
+final locator = GetIt.instance;
```

Comment:
I'm not sure if this is necessary as I saw you already created an instance of it below? 😓

Reply:
I just want to create a global variable so I can use like these everywhere
```dart
class ApiServiceImpl implements ApiService {
  static String? _fallbackBaseUrl;
  static String? _fallbackToken;
  final HttpService _httpService = locator.get();
  ...
```

Reply:
@markgravity I'm curious too. Can't we use `locator` variable below as a global variable? 🙏 Creating one more variable to hold GetIt instance is just wasting memory 🤔

Reply:
@sleepylee , @manh-t my bad. It's redundant code line 😁

Reply:
> @markgravity I'm curious too. Can't we use `locator` variable below as a global variable? 🙏 Creating one more variable to hold GetIt instance is just wasting memory 🤔

This `locator` there is a reference to `GetIt.instance`, so I think it won't waste the memory IMO 😀